datadog_lambda/tracing.py

```diff
@@ -101,14 +134,18 @@ def extract_dd_trace_context(event):
     the correct context.
     """
     global dd_trace_context
-    headers = event.get("headers", {})
-    lowercase_headers = {k.lower(): v for k, v in headers.items()}
 
-    trace_id = lowercase_headers.get(TraceHeader.TRACE_ID)
-    parent_id = lowercase_headers.get(TraceHeader.PARENT_ID)
-    sampling_priority = lowercase_headers.get(TraceHeader.SAMPLING_PRIORITY)
+    if "headers" in event:
+        trace_id, parent_id, sampling_priority = extract_context_from_http_event(event)
+    elif "Records" in event:
+        trace_id, parent_id, sampling_priority = extract_context_from_sqs_event(event)
```

Comment:
If "Records" is in the event, but we fail to parse, we should probably try extracting from the lambda context next.

Reply:
Oh, whoops, didn't see Tian's review. Looks like he already mentioned this

Reply:
Use the lambda context as a fallback for both `headers` and `SQS` events